### PR TITLE
JBPM-7886 : Update WildFly Swarm to Thorntail

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/JBPM-7886

@tomasdavidorg Making draft PRs for tests

Bundle:
https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/944
https://github.com/kiegroup/kie-wb-common/pull/2581
https://github.com/kiegroup/kie-wb-distributions/pull/906

### CODE REVIEW

**Comment by tomasdavidorg** [comment]
Jenkins execute full downstream build

**Comment by ge0ffrey** [comment]
April fools? :)

**Comment by Rikkola** [comment]
@ge0ffrey lol, nope.

**Comment by Rikkola** [comment]
By the way it might make sense to update Wildfly to the same level, but will need to see about that. I'm getting few dependency issues on Thorntail jar startup.

**Comment by Rikkola** [comment]
It takes forever to run the build for this, so I'll just slowly make changes and use the jenkins to do my builds ;)

Jenkins execute full downstream build

**Comment by MarianMacik** [comment]
Jenkins execute full downstream build

File: pom.xml [hunk 1]
```diff
@@ -129,7 +129,7 @@
     <version.org.jboss.jboss-dmr>1.5.0.Final</version.org.jboss.jboss-dmr>
     <!-- Version of Camel patch for Wildfly -->
     <version.org.wildfly.camel>8.0.0</version.org.wildfly.camel>
-    <!-- this version will overwrite jboss-ip-bom version 2.5.1. The version in jboss-ip-bom couldn't be upgraded as it will break ModeShape Web Explorer application which uses GWT 2.5.1 -->
+    <!-- this version will overwrite jboss-ip-bom version 2.5.1. The version in jboss-ip-bom couldn't be upgraded as it will break ModeShape Web Explorer applicaill break ModeShape Web Ex-->
```
Comment (mareknovotny) [inline pom.xml]: is this change really intended?

**Comment by mareknovotny** [comment]
Jenkins execute full downstream build

File: pom.xml (hunk 1 above)
Comment (Rikkola) [reply pom.xml]: Nope. Not sure how it even got here.
